pkgs/tools/X11/wpgtk/default.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, python3, python36Packages, fetchFromGitHub, pywal, feh, libxslt, imagemagick,
+  gobjectIntrospection, gtk3, wrapGAppsHook, gnome3 }:
+
+python36Packages.buildPythonApplication rec {
+  name = "${pname}-${version}";
+  pname = "wpgtk";
+  version = "5.7.4";
+
+  src = fetchFromGitHub {
+    owner = "deviantfero";
+    repo = "wpgtk";
+    rev = "${version}";
+    sha256 = "0c0kmc18lbr7nk3hh44hai9z06lfsgwxnjdv02hpjwrxg40zh726";
+  };
+
+  pythonEnv = python3.withPackages(ps: with ps; [ pygobject3 pillow pip]);
+
+  propagatedBuildInputs = [
+    wrapGAppsHook
+    gtk3
+    gobjectIntrospection
+    gnome3.adwaita-icon-theme
+    pythonEnv
+    pywal
+    feh
```

Comment:
Adding programs to inputs will most likely not be enough. See https://stackoverflow.com/a/34837585/160386 for a description of how runtime dependencies work.

Reply:
I added them to `buildInputs` respectively `pythonPath` and it worked when I tested it. `feh` is being used by the program on runtime and I tested that successfully (feh isn't installed in my environment). Can I assume that this is not an issue or is there a way to test it?

Reply:
Grepping the source code, `feh` is not even used.